e2etest/bgp_test.go

```diff
@@ -955,3 +1030,19 @@ func uint32Ptr(n uint32) *uint32 {
 func boolPtr(b bool) *bool {
 	return &b
 }
+
+func frrIsCommunitySet(cs clientset.Interface, n *frrcontainer.FRR) {
+	pods := getSpeakerPods(cs)
+	podExecutor := executor.ForPod(testNameSpace, pods[0].Name, "frr")
```

Comment:
I would check this in the external containers rather than in the pods.

Also, I think the community-info part (by parsing the result) belongs here: https://github.com/metallb/metallb/blob/main/e2etest/pkg/frr/bgp.go